crates/iota-indexer/src/apis/indexer_api.rs

```diff
@@ -248,7 +249,8 @@ impl IndexerApiServer for IndexerApi {
         )
         .expect("deriving dynamic field id can't fail");
 
-        let options = iota_json_rpc_types::IotaObjectDataOptions::full_content();
+        let options = options.unwrap_or_else(IotaObjectDataOptions::full_content);
```

Comment:
For get_object() it just uses `let options = options.unwrap_or_default();` which has set all the options to false, just wondering if it would make more sense to do the same here? Otherwise default will require much more data

Reply:
I see what you mean but then we have a breaking change. Each dep that makes use of the current `get_dynamic_field_object` function expects the full content as that was the default behavior for this function before. Shouldn't we avoid changing that? What do you think? @Thoralf-M @thibault-martinez

Reply:
Ah ok wasn't aware that it was like that before also. So it returns everything except bcs and display, I thought full_content also includes these
Then it should be fine as it is :+1: